Gemfile

```diff
@@ -67,7 +67,8 @@ else
 end
 
 gem 'sass', '3.4.9'
-gem 'sass-rails'
+gem 'sass-rails' # required by bootstrap-sass, but not listed as a dependency of it
+gem 'sassc-rails', github: 'bolandrm/sassc-rails', branch: 'master'
```

Comment:
Yeah, hopefully this is a short-term thing, the gem should be released at some point soon at which point we can point it at a release.